### PR TITLE
uint32->uint64 for RedHat gflags 2.1 failing at runtime

### PR DESCRIPTION
**Pull request by Wovchena**
Back up plan. Do not merge yet

### CODE REVIEW

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 1]
```diff
@@ -14,15 +14,6 @@
 
 #include "gflags/gflags.h"
 
-// gflags supports uint32 starting from v2.2 only
```
Comment (ilyachur) [inline samples/cpp/benchmark_app/benchmark_app.hpp:17]: What if user used system gflags which doesn't support uint32...

Comment (Wovchena) [reply samples/cpp/benchmark_app/benchmark_app.hpp:17]: That's why I use uint64

**Comment by ilya-lavrenov** [comment]
> Back up plan. Do not merge yet

Can we merge it now?
We are still required to build against old gflags versions

**Comment by Wovchena** [comment]
Ok

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 2]
```diff
@@ -14,15 +14,6 @@
...
-#ifndef DEFINE_uint32
-#    ifdef GFLAGS_NAMESPACE
-#        define DEFINE_uint32(name, val, txt) DEFINE_VARIABLE(GFLAGS_NAMESPACE::uint32, U, name, val, txt)
-#    else
-#        define DEFINE_uint32(name, val, txt) DEFINE_VARIABLE(gflags::uint32, U, name, val, txt)
```
Comment (ilya-lavrenov) [inline samples/cpp/benchmark_app/benchmark_app.hpp:22]: can we also remove such WA from other places in repo?

Comment (Wovchena) [reply samples/cpp/benchmark_app/benchmark_app.hpp:22]: Removed

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
Please, port to 2023.0

**Pull request merged**